components/Sidebar/panels/Playback/Playback.tsx

```diff
@@ -0,0 +1,148 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import { useState, useTransition } from "react";
+
+import getCurrentTrackUri from "@/lib/playback/gettrackrui";
+
+import play from "../../../../lib/playback/play";
+import next from "../../../../lib/playback/skipnext";
+import prev from "../../../../lib/playback/skipprev";
```

Comment:
Can you make an index file inside of playback and export all of these from there? That way, you can do `import { play, next, prev } from "@/lib/playback";` ?